contrib/kafka/pkg/controller/channel/reconcile_test.go

```diff
@@ -30,10 +30,12 @@ import (
 	eventingv1alpha1 "github.com/knative/eventing/pkg/apis/eventing/v1alpha1"
 	"github.com/knative/eventing/pkg/provisioners"
 	util "github.com/knative/eventing/pkg/provisioners"
+	"github.com/knative/eventing/pkg/reconciler/names"
 	controllertesting "github.com/knative/eventing/pkg/reconciler/testing"
 	"github.com/knative/eventing/pkg/utils"
 	duckv1alpha1 "github.com/knative/pkg/apis/duck/v1alpha1"
 	istiov1alpha3 "github.com/knative/pkg/apis/istio/v1alpha3"
```

Comment:
this still is on istio:  

https://github.com/knative/eventing/blob/09e4dfa03adc27e7bf516a3bddb5fcd98d2e71e7/contrib/kafka/pkg/controller/channel/reconcile_test.go#L75